pages/common/coproc.md

```diff
@@ -0,0 +1,28 @@
+# coproc
+
+> BASH builtin for creating interactive asynchronous subshells.
```

Comment:
Although we have a precedent for `BASH` being in all caps for `common/read` & `linux/jobs`, should we keep it as `ALL-CAPS` ?
@EmilySeville7cfg 
[Apologies if there was discussion about it already]

Reply:
A quick grep of the English pages shows that all-caps `BASH` is used much less often than `Bash`, `common/read` just happened to be the Bash builtin page I used as a reference. I'd be happy to change it, especially since I prefer `Bash` myself.

Reply:
> Although we have a precedent for BASH being in all caps for common/read & linux/jobs, should we keep it as ALL-CAPS ?

I don't think so, but this my opinion. Wikipedia page uses `Bash` for instance. As there is no rule requiring to use `BASH` or `Bash` it's up to @FMakosza. But all pages should have consistent Bash naming.